test/cmdlineTests/ast_compact_json_with_base_path/output

```diff
@@ -45,7 +45,8 @@ JSON AST (compact format):
       "nodes": [],
       "scope": 6,
       "src": "60:13:0",
-      "usedErrors": []
+      "usedErrors": [],
+      "usedEvents": []
```

Comment:
Is there a test where this list actually contains something?

Reply:
Yes, there are a few. Those are the ones with huge green diffs. They were added specifically because the existing tests did not have enough coverage for what this PR does.